### PR TITLE
Added How-to-use instructions

### LINKED ISSUE #7: Add how-to-use instructions
Add how-to-use instructions to ``README.md``.

### PR DESCRIPTION
**Pull request by pwilkins**
#### What are the relevant tickets?
Fixes #7

#### What's this PR do?
Provides information about running the release-script.  

#### Where should the reviewer start?
Consider whether there is sufficient information provided to run the script
and understand the result.

#### How should this be manually tested?
Simply run the script.

#### Any background context you want to provide?
The last few steps are not yet implemented.

#### What GIF best describes this PR or how it makes you feel?

mouse under treadwheel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,44 @@
...
-8. Generate release notes with checkboxes
-8. Open PR against ``release`` branch
-10. Open PR against ``master`` branch
+1. Check-out a current ``master`` branch
+2. Create a ``release-candidate`` branch
+3. Hard reset ``release-candidate`` to ``master``
+4. Generate release notes
+5. Update version numbers and ``RELEASE.rst``
+6. Commit updates and push ``release-candidate`` branch
+7. Open PR against ``release-candidate`` branch
```
Comment (justinabrahms) [inline README.md]: This doesn't happen.

File: README.md [hunk 2]
```diff
@@ -4,21 +4,44 @@
...
+8. Merge PR once ``travis-ci`` build succeeds
```
Comment (justinabrahms) [inline README.md]: Nor this.

File: README.md [hunk 3]
```diff
@@ -4,21 +4,44 @@
...
+9. Generate release notes with checkboxes
+10. Open PR against ``release`` branch
 11. Merge PRs once developers verify their commits
-12. Send email notifications
+12. Open PR to merge ``release`` into ``master`` branch
```
Comment (justinabrahms) [inline README.md]: I think this should just be a direct merge and push, not a PR.

File: README.md [hunk 4]
```diff
@@ -4,21 +4,44 @@
...
+13. Send email notifications
```
Comment (justinabrahms) [inline README.md]: Missing tagging the build.

**Comment by gsidebo** [comment]
👍

**Pull request merged**